tests/golden.csv

```diff
@@ -2941,7 +3207,7 @@ Selina Ho,theendtimenews.com,2020-06-01,https://theendtimenews.com/,FALSE,,,,
 Selina Ho,theendtimenews.com,2020-06-01,https://theendtimenews.com/2020/,FALSE,,,,
 Selina Ho,theendtimenews.com,2020-06-01,https://theendtimenews.com/2020/04/,FALSE,,,,
 Selina Ho,theendtimenews.com,2020-06-01,https://theendtimenews.com/2020/04/21/,FALSE,,,,
-Selina Ho,theendtimenews.com,2020-06-01,https://theendtimenews.com/2020/04/21/heart-surgery-kim-jong-uns-health-is-serious/,TRUE,en-us,2020-04-21 06:10:12+00:00,2020-04-21 06:10:12+00:00,Heart surgery; Kim Jong Un’s health is serious
+Selina Ho,theendtimenews.com,2020-06-01,https://theendtimenews.com/2020/04/21/heart-surgery-kim-jong-uns-health-is-serious/,TRUE,en-us,2020-04-21 06:10:12+00:00,2020-04-21 06:10:12+00:00,Heart surgery, Kim Jong Un’s health is serious
```

Comment:
There's a few places where you've changed other people's titles by replacing `;` with `,`.  I assume this was an accident, and you didn't intend to, but do you happen to have any idea what part of your workflow is causing this?

Reply:
I was having some errors with the ValueError regarding an extra comma, so I ran the following comman
```
sed -i 's/,$//' tests/golden.csv
```

Perhaps this did something? But apart from that, I didn't do anything to other people's titles.

Reply:
That command won't cause this.  Are you using excel to open the file?  My guess is that it is interpreting both `;` and `,` as delimiters, and so when it saves it replaces all the `;` with `,`.  This would also cause the problem of the `,` at the end of the lines since it would cause excel to think there is an additional column.

Reply:
No, I have only been using LibreOffice. I'm not sure what that seems to be happening.